src/Maker/MakeMigration.php

```diff
@@ -0,0 +1,102 @@
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Doctrine\Bundle\MigrationsBundle\Command\DoctrineCommand;
+use Symfony\Bundle\MakerBundle\ApplicationAwareMakerInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\ExtraGenerationMakerInterface;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\MakerInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+final class MakeMigration implements MakerInterface, ApplicationAwareMakerInterface, ExtraGenerationMakerInterface
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    public static function getCommandName(): string
+    {
+        return 'make:migration';
+    }
+
+    public function setApplication(Application $application)
+    {
+        $this->application = $application;
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command
+            ->setDescription('Creates a new migration based on database changes.')
+            ->addOption('db', null, InputOption::VALUE_REQUIRED, 'The database connection to use for this command.')
+            ->addOption('em', null, InputOption::VALUE_OPTIONAL, 'The entity manager to use for this command.')
+            ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command.')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeMigration.txt'))
+        ;
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+    }
+
+    public function getParameters(InputInterface $input): array
+    {
+        return array(
+            'options' => array(
+                'db' => $input->getOption('db'),
+                'em' => $input->getOption('em'),
+                'shard' => $input->getOption('shard'),
+            ),
+        );
+    }
+
+    public function getFiles(array $params): array
+    {
+        return array();
+    }
+
+    public function afterGenerate(ConsoleStyle $io, array $params)
+    {
+        $options = $params['options'];
+
+        $options['command'] = 'doctrine:migrations:diff';
+        $generateMigrationCommand = $this->application->find('doctrine:migrations:diff');
+        $generateMigrationCommand->run(new ArgvInput($options), $io);
+    }
+
+    public function writeNextStepsMessage(array $params, ConsoleStyle $io)
+    {
+        $io->text(array(
+            'Next: Edit Version(lastVersion) to customize the migration.',
+            'Find the documentation at <fg=yellow>https://symfony.com/doc/current/bundles/DoctrineMigrationsBundle/index.html</>',
```

Comment:
Could we standardize these doc links in all commands? `Find the documentation at...` looks too long to me. What about `See ...` ?